src/frontend/apps/web/src/index.scss

```
@@ -6,6 +6,8 @@
 @import "./components/Header";
 @import "./components/SelectContent";
 
+@import "ui/index";
```

Check my comment related to renaming the `ui/index.scss` stylesheet.

```suggestion
@use "ui"
```